.github/workflows/equinix_metal_flow.yml

```diff
@@ -0,0 +1,149 @@
+name: Equinix Metal Action
+
+on:
+  workflow_dispatch:
+
+jobs:
+  Create-runner:
+    name: "Create Runner"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: metal-runner-action
+        uses: rootfs/metal-runner-action@main
+        with:
+          github_token: ${{ secrets.GH_SELF_HOSTED_RUNNER_TOKEN }}
+          metal_auth_token: ${{ secrets.EQUINIX_API_TOKEN }}
+          metal_project_id: ${{ secrets.EQUINIX_PROJECT_ID }}
+          metro: "da"
+          plan: "c3.small.x86"
+          os: "rhel_9"
+
+  Install:
+    name: "Install"
+    needs: Create-runner
+    runs-on: self-hosted
+    continue-on-error: true
+    outputs:
+      runner-name: ${{ runner.name }}
+
+    steps:
+      - run: |
+          echo "This is runner: ${{ runner.name }}"
```

Comment:
In a different PR, the code could be moved to a bash script so that it can executed manually on a manually provisioned machine.

Reply:
+1 this script or ansible approach should be sharable or reusable?